lib/stronger_parameters/parameters.rb

```diff
@@ -56,11 +56,11 @@ def integer64
       end
 
       def id
-        integer & lte(2 ** 31) & gte(0)
+        integer & gte(0)
       end
 
-      def bigid
-        integer & lte(2 ** 63) & gte(0)
+      def smallid
+        integer & lte(2 ** 32) & gte(0)
```

Comment:
for unsigned 32-bit integer: `lte(2 ** 32)` -> `lt(2 ** 32)`

Range is 0 to 4294967295 ((2^32) - 1)